antarest/study/storage/study_upgrader/__init__.py

```diff
@@ -196,6 +197,9 @@ def _copies_only_necessary_files(files_to_upgrade: t.List[Path], study_path: Pat
     files_to_retrieve = []
     for path in files_to_copy:
         entire_path = study_path / path
+        if not entire_path.exists():
```

Comment:
Je ne comprends pas bien dans quel cas cela se produit.

Reply:
I made a little explanation in the upgrader 8.8 file. I believe it can happen when upgrading a study without st storages to v8.8.